ext/node/polyfills/internal/fs/handle.ts

```diff
@@ -148,10 +156,17 @@ export class FileHandle extends EventEmitter {
   stat(options?: { bigint: boolean }): Promise<Stats | BigIntStats> {
     return fsCall(promises.fstat, this, options);
   }
+
   chmod(mode: Mode): Promise<void> {
     assertNotClosed(this, promises.chmod.name);
     return promises.chmod(this.#path, mode);
   }
+
+  createReadStream(options?: CreateReadStreamOptions): ReadStream {
+    assertNotClosed(this, createReadStream.name);
```

Comment:
This assertion doesn't exist in Node.js https://github.com/nodejs/node/blob/5770972dc6c0458af5458b8056b16d70905ea9d8/lib/internal/fs/promises.js#L361-L364